<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QWidget" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/sourse/stack_1.jpg);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>220</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-image: url(:/sourse/stack_2.jpg);
font: 36pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/sourse/stack_2.jpg);
font: 36pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>游戏介绍</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/sourse/stack_2.jpg);
font: 36pt &quot;华文新魏&quot;;
</string>
   </property>
   <property name="text">
    <string>游戏设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>600</y>
     <width>300</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文新魏</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/sourse/stack_2.jpg);
font: 36pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>离开游戏</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 100pt &quot;方正字迹-少壮简体&quot;;
background: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>滴答</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
